src/server/pbsd_init.c

```diff
@@ -317,6 +318,16 @@ init_server_attrs()
 	attr_jobscript_max_size.at_type  |= ATR_TYPE_SIZE;  /* get_bytes_from_attr() is checking for at_type */
 	set_size(&attr_jobscript_max_size,&attrib,SET);
 
+	sprintf(dflt_log_event, "%d", PBSEVENT_ERROR | PBSEVENT_SYSTEM | PBSEVENT_ADMIN | PBSEVENT_JOB | PBSEVENT_JOB_USAGE |
+			PBSEVENT_SECURITY | PBSEVENT_SCHED | PBSEVENT_DEBUG | PBSEVENT_DEBUG2);
+	set_attr_svr(&(server.sv_attr[(int)SRV_ATR_log_events]), &svr_attr_def[(int) SRV_ATR_log_events], dflt_log_event);
+
+	set_attr_svr(&(server.sv_attr[(int)SRV_ATR_mailfrom]), &svr_attr_def[(int) SRV_ATR_mailfrom], "adm");
+
+	set_attr_svr(&(server.sv_attr[(int)SRV_ATR_query_others]), &svr_attr_def[(int) SRV_ATR_query_others], "TRUE");
+
+	set_attr_svr(&(server.sv_attr[(int)SRV_ATR_scheduling]), &svr_attr_def[(int) SRV_ATR_scheduling], "TRUE");
```

Comment:
Use ATR_TRUE.  It's an include in pbs_internal.h.  Use ATR_TRUE/ATR_FALSE in other places in the code. 
 Also, did you test that the -a 0 option still works that this doesn't override it?

Reply:
I tested it with -a0 option and it does set scheduling to false.